### PR TITLE
Update forking_demo

### PR DESCRIPTION
**Pull request by aestopettyy**
ok

### CODE REVIEW

**Comment by aestopettyy** [comment]
ok

File: forking_demo [hunk 1]
```diff
@@ -2,5 +2,8 @@ jo bhi meri file me forking kr raha h
 koi bhi line delete kr doo aur koi bhi ek line add kr doo.
 necche di gyi line me forking karo.
 
-My name is Prakhar Gupta
-i study in NIET Collage
+
+i study in IILM Collage
```
Comment (kenma08) [inline forking_demo:6]: Ye q kiya

File: forking_demo [hunk 2]
```diff
@@ -2,5 +2,8 @@ jo bhi meri file me forking kr raha h
 koi bhi line delete kr doo aur koi bhi ek line add kr doo.
 necche di gyi line me forking karo.
 
-My name is Prakhar Gupta
-i study in NIET Collage
```
Comment (kenma08) [inline forking_demo:6]: Sach Me

File: forking_demo (hunk 1 above)
Comment (aestopettyy) [reply forking_demo:6]: mera mnn

File: forking_demo (hunk 2 above)
Comment (aestopettyy) [reply forking_demo:6]: mera mnn

**Pull request merged**